<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arandu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo_Projeto"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionFechar_Projeto"/>
    <addaction name="actionCompactar"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConfigura_es">
    <property name="title">
     <string>Configurações</string>
    </property>
    <addaction name="actionSQL_Projeto"/>
    <addaction name="actionResetar_Layout"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionContato"/>
    <addaction name="actionFAQ"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuConfigura_es"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QDockWidget" name="conteudo">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>141</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Conteúdo</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTreeView" name="content"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="navegacao">
   <property name="windowTitle">
    <string>Navegação</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeView" name="nav_folder">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="modelos">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>517</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Modelos</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <property name="tabSpacing">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>240</width>
          <height>228</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Divisão de Dados</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_AA">
           <property name="text">
            <string>Amostragem Aleatória</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Conjunto de Treinamento (%):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>70</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Conjunto de Testes (%):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_2">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_DD">
           <property name="text">
            <string>Dados Totais</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>232</width>
          <height>191</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Modelo ML</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="checkBox_RF">
           <property name="text">
            <string>Random Forest</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_RN">
           <property name="text">
            <string>Redes Neurais</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <attribute name="label">
         <string>Arquitetura</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>N.º Camadas Ocultas:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinbox_camadas_ocultas">
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>N.º Neurônios / Camada</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>N.º Camadas Dropout:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinbox_camadas_dropout">
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Valor Dropout / Camada</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QFrame" name="holder_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2"/>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Função de Saída:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="comboBox">
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Quantidade de Métricas:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="spinBox_3"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QFrame" name="holder">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout"/>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <attribute name="label">
         <string>Otimizadores</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_5">
        <attribute name="label">
         <string>Loss</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_6">
        <attribute name="label">
         <string>Call Backs</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_7">
        <attribute name="label">
         <string>Configuração de Ajuste</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNovo_Projeto">
   <property name="text">
    <string>Novo Projeto</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir Projeto</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionFechar_Projeto">
   <property name="text">
    <string>Fechar Projeto</string>
   </property>
  </action>
  <action name="actionCompactar">
   <property name="text">
    <string>Compactar</string>
   </property>
  </action>
  <action name="actionAbrir_Arquivo">
   <property name="text">
    <string>Abrir Arquivo</string>
   </property>
  </action>
  <action name="actionRedes_Neurais">
   <property name="text">
    <string>Redes Neurais</string>
   </property>
  </action>
  <action name="actionRandom_Forest">
   <property name="text">
    <string>Random Forest</string>
   </property>
  </action>
  <action name="actionSQL_Projeto">
   <property name="text">
    <string>SQL Projeto</string>
   </property>
  </action>
  <action name="actionResetar_Layout">
   <property name="text">
    <string>Resetar Layout</string>
   </property>
  </action>
  <action name="actionContato">
   <property name="text">
    <string>Contato</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>FAQ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
